check_duplicates.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+files_map = {}
+
+for root, dir, files in os.walk("src/catch2"):
+    for file in files:
+        if file not in files_map:
+            files_map[file] = 1
```

Comment:
Why not use a `set` and not have to make up a dummy key?

Alternatively, keep around the full path to the file, so the error can also report paths to the file... (But using plain set is simpler).

Reply:
I did not know python had sets thanks for telling me.